docs/installation/setup.rst

```diff
@@ -11,131 +11,137 @@ Prerequisites
 .. _SaltStack: https://www.saltstack.com
 .. _Puppet: https://puppet.com
 
-MetalK8s_ clusters require machines running CentOS_ / RHEL_ 7.6 or higher as
+MetalK8s_ clusters require machines running CentOS_//RHEL_ 7.6 or higher as
 their operating system. These machines may be virtual or physical, with no
-difference in setup procedure. The number of machines to setup depends on the
-chosen architecture (see :ref:`installation-intro-architecture`).
+difference in setup procedure. The number of machines to set up depends on the
+architecture you chose in :ref:`installation-intro-architecture`.
 
-Machines must **not** be managed by any configuration management system
-(e.g. SaltStack_, Puppet_).
+Machines *must not* be managed by any configuration management system, such as
+SaltStack_ or Puppet_.
 
     .. warning::
-        Distribution must be, as much as possible, left intact (no tuning,
-        tweaking, configuration nor software installation).
+       The distribution must be left intact as much as possible (do not tune,
+       tweak, or configure it, or install any software).
 
 Proxies
 -------
-For nodes operating behind a proxy, see :ref:`Bootstrap Configuration`
+
+For nodes operating behind a proxy, see :ref:`Bootstrap Configuration`.
 
 Linux Kernel Version
 --------------------
-Linux Kernel shipped with latest (7.7) and previous versions of
-CentOS_ / RHEL_ 7 is affected by a cgroups memory leak bug.
 
-Kernel must be at least in version 3.10.0-1062.4.1 for this bug to be fixed.
+Linux kernels shipped with CentOS/RHEL 7 and earlier are affected by a
+cgroups memory leak bug.
+
+This bug was fixed in kernel 3.10.0-1062.4.1. Use this kernel version or later.
 
 The version can be retrieved using:
 
-    .. code-block:: shell
+  .. code-block:: shell
 
-        uname -r
+    $ uname -r
 
-If the installed version is lower than the one above, it must
-be upgraded:
+If the installed version is lower than the one above, upgrade it with:
 
-    .. code-block:: shell
+.. code-block:: shell
```

Comment:
Inconsistent use of indentation compared to the change above.

Reply:
fixed